editor/scene/3d/node_3d_editor_plugin.cpp

```diff
@@ -861,20 +862,20 @@ ObjectID Node3DEditorViewport::_select_ray(const Point2 &p_pos) const {
 				continue;
 			}
 
-			const real_t dist = pos.distance_to(point);
+			const real_t dist_squared = pos.distance_squared_to(point);
 
-			if (dist < 0) {
+			if (dist_squared < 0) {
```

Comment:
There are a few cases like this across the code base that make no sense, didn't change it in this case but would consider removing them as they are never true